### ISSUE TITLE
Add Swedish municipalities

### ISSUE THREAD

**Issue by migurski** [whosonfirst-data/whosonfirst-data#617]
According to Wikipedia, [there are 290 Swedish municipalities](https://en.wikipedia.org/wiki/List_of_municipalities_of_Sweden). An example is [Höganäs](https://whosonfirst.mapzen.com/spelunker/id/101816585/#11/56.2093/12.5643), which WoF has as a locality (pop. 13k) but not as a municipality (pop. 25k). https://github.com/openaddresses/openaddresses/pull/2350 includes data for the municipality, so I’m  thinking it’d be useful to have this level represented.

**Comment by nvkelso** [comment]
Agree these should be added, thanks for the tip!

When we blogged about the [mesoshapes import](https://mapzen.com/blog/mesoshapes/) the Europe map didn't have much in the way of `county` coverage. This was due to two factors: 

1. Europe seems to jump to `localadmin` from `region` in many countries, skipping over `county`. 
2. 5 years ago Europe was still getting on the open data bandwagon. 

We'll need to determine if the municipalities proposed here are `localadmin` or `county`, and find a source.
